current_fall2025/scripts/run_queries.py

```
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)